aiven/client/cli.py

```diff
@@ -3059,6 +3059,93 @@ def mirrormaker__replication_flow__delete(self):
             self.args.target_cluster,
         )
 
+    @arg.project
+    @arg.service_name
+    def service__flink__table__list(self):
+        """List Flink tables"""
+        project_name = self.get_project()
+        self.print_response(self.client.list_flink_tables(project_name, self.args.service_name))
```

Comment:
Now this outputs JSON by default. Comparing to others list commands of `avn` I think this should have `@arg.json` and by default print human readable table. Same goes for flink table get